src/js/tables.js

```diff
@@ -1,11 +1,14 @@
-/*global require*/
 /**
  * Wrap tables
  */
-'use strict';
 
-var $ = require('jquery');
+'use strict';
 
-$(function() {
-	$('.o-techdocs-content > table').wrap('<div class="o-techdocs-table-wrapper"></div>');
-});
+document.addEventListener('o.DOMContentLoaded', function() {
+	Array.from(document.querySelectorAll('.o-techdocs-content > table')).forEach(function(el) {
+		var wrapper = document.createElement('div');
+		wrapper.className = "o-techdocs-table-wrapper";
```

Comment:
I might start using this: 'http://ariya.ofilabs.com/2012/02/from-double-quotes-to-single-quotes.html' 😅